e2e_samples/dataset_versioning/datafactory/arm_template/arm_template.json

```diff
@@ -0,0 +1,311 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "factoryName": {
+            "type": "string",
+            "metadata": "Data Factory name",
+            "defaultValue": "adf-loan-dev"
+        },
+        "KeyVault_properties_typeProperties_baseUrl": {
+            "type": "string",
+            "defaultValue": "https://kv-loan-dev-eastus.vault.azure.net/"
+        },
+        "AzureSqlDatabase_properties_typeProperties_connectionString_secretName": {
+            "type": "string",
+            "defaultValue": "watermarkdb-connection"
+        },
+        "AzureBlobFS_properties_typeProperties_serviceEndpoint": {
+            "type": "string",
+            "defaultValue": "https://dlsloandev.blob.core.windows.net/"
```

Comment:
This isn't in Key Vault?

Reply:
I've deleted `defaultvalue` here

> This isn't in Key Vault?
Yes, we don't have key vault for this. Because we are using managed identity for connecting from SQL to Storage, we just need url for storage account. I'll ask an end users to input storage endpoint as parameter via azure cli when they deploy ARM template.

Reply:
Ah OK my mistake

Reply:
No, it's not mistake. I needed to write background.
Thank you for your comment.
Resolved